.gitignore

```diff
@@ -26,8 +26,10 @@ dist-ssr
 
 # artifacts
 scrapers/catalog.json
-scrapers/fireroad.json
+scrapers/fireroad-sem.json
+scrapers/fireroad-presem.json
 public/latest.json
+public/i25.json
```

Comment:
Is there no other way of making sure we don't commit presemester? I feel like this will start getting bloated after a few semesters

Reply:
The main other way that I can think of is to put the latest pre-semester in `public/latestPreSemester.json` and then to add a special case to `src/lib/TermSwitcher.tsx` so that the `urlName` corresponding to the latest pre-semester will read from the file `public/latestPreSemester.json`.

This would break the correspondence between `urlName`s and JSON file names that we have now; e.g., `https://hydrant.mit.edu/?t=latest` loads Spring 2025 while `https://hydrant.mit.edu/?t=s25` does not, since the file is `latest.json` and not yet `s25.json`.

Alternatively, we could make `public/i25.json` a symlink to `public/latestPreSemester.json`, but then the maintenance task on each semester is changed from editing `.gitignore` to modifying a symlink (which is IMO marginally worse).

Bloat shouldn't become an issue here, since it's only the latest pre-semester that needs to be ignored; when the fall 2025 listings are out, `public/i25.json` should be committed (along with `public/s25.json`) and it will be `public/m25.json` that will need to be ignored.